bio_hansel/qc/checks.py

```diff
@@ -132,18 +129,16 @@ def is_mixed_subtype(st: Subtype, df: DataFrame, *args) -> Tuple[Optional[str],
     """
     if not st.are_subtypes_consistent:
         return QC.FAIL, \
-               '{}: Mixed subtypes found: "{}".'.format(
-                   QC.MIXED_SAMPLE_ERROR_2,
+               'Mixed subtypes found: "{}".'.format(
                    '; '.join(sorted(st.inconsistent_subtypes)))
     conflicting_tiles = get_conflicting_tiles(st, df)
     if conflicting_tiles is None or conflicting_tiles.shape[0] == 0:
         return None, None
 
-    return QC.FAIL, '{}: Mixed subtype detected. Positive and negative tiles detected for ' \
-                    'the same target site "{}" for subtype "{}".'.format(
-                        QC.MIXED_SAMPLE_ERROR_2,
-                        '; '.join(conflicting_tiles['refposition'].astype(str).tolist()),
-                        st.subtype)
+    return QC.FAIL, ('Mixed subtype detected. Positive and negative tiles detected for '
+                     'the same target site "{}" for subtype "{}".').format(
```

Comment:
Maybe make the message more clear for multiple target sites? "Positive and negative tiles detected for the following target sites: 282929, 228292, 2982822 for subtype 1, 2, 3" to make it easier for the user?

Reply:
That QC message could definitely be clearer and there could definitely be multiple target sites where the positive and negative tiles are present. I'm wondering how to make this message more informative for non-experts to explain why it's not ideal to have both positive and negative tiles present at the same location.